main.ts

```diff
@@ -1 +1,3 @@
-import "cli/main";
+import "tsconfig-paths/register";
```

Comment:
NOTE: This is necessary to resolve paths correctly on my end. Not yet sure why we did not need it before 🤷‍♀️

Reply:
what are you using to run it?

Reply:
Happens both with `ts-node` and `tsx`. Without it, it complains about `react` not existing etc. (because we map it in `tsconfig`'s `paths`).